easybuild/easyconfigs/q/Qt5/Qt5-5.10.1_glibc-fix.patch

```diff
@@ -0,0 +1,26 @@
+Fix Qt5.10.1 to build against newer qlibc.
```

Comment:
@branfosj Let's fix the `qlibc` -> `glibc` typo here? ;)

Reply:
I made the same typo in the subject line :(

However, I hit a further issue in the build:
```
FAILED: gn
/rds/bear-apps/devel/eb-sjb-up/EL8/EL8-cas/software/Python/2.7.15-foss-2018b/bin/python2 "../../../../build/toolchain/gcc_link_wrapper.py" --output="./gn" -- /rds/bear-apps/devel/eb-sjb-up/EL8/EL8-cas/software/GCCcore/7.3.0/bin/g++ -Wl,--fatal-warnings -fPIC -Wl,-z,noexecstack -Wl,-z,now -Wl,-z,relro -Wl,-z,defs -Wl,--no-as-needed -lpthread -Wl,--as-needed -fuse-ld=gold -B../../../../third_party/binutils/Linux_x64/Release/bin -m64 -Wl,-O1 -Wl,--gc-sections -Wl,-rpath-link=. -Wl,--disable-new-dtags -o "./gn" -Wl,--start-group @"./gn.rsp"  -Wl,--end-group   -ldl -lpthread -lrt
/lib/../lib64/crti.o(.gnu.build.attributes+0x14): warning: relocation refers to discarded section
/lib/../lib64/crti.o(.gnu.build.attributes+0x1c): warning: relocation refers to discarded section
/lib/../lib64/crtn.o(.gnu.build.attributes+0x14): warning: relocation refers to discarded section
/lib/../lib64/crtn.o(.gnu.build.attributes+0x1c): warning: relocation refers to discarded section
/lib/../lib64/crt1.o(.gnu.build.attributes.hot+0x18): warning: relocation refers to discarded section
/lib/../lib64/crt1.o(.gnu.build.attributes.hot+0x20): warning: relocation refers to discarded section
/lib/../lib64/crt1.o(.gnu.build.attributes.unlikely+0x18): warning: relocation refers to discarded section
/lib/../lib64/crt1.o(.gnu.build.attributes.unlikely+0x20): warning: relocation refers to discarded section
/lib/../lib64/crt1.o(.gnu.build.attributes.startup+0x18): warning: relocation refers to discarded section
/lib/../lib64/crt1.o(.gnu.build.attributes.startup+0x20): warning: relocation refers to discarded section
/lib/../lib64/crt1.o(.gnu.build.attributes.exit+0x18): warning: relocation refers to discarded section
/lib/../lib64/crt1.o(.gnu.build.attributes.exit+0x20): warning: relocation refers to discarded section
/lib/../lib64/crt1.o(.gnu.build.attributes.hot+0x18): warning: relocation refers to discarded section
/lib/../lib64/crt1.o(.gnu.build.attributes.hot+0x20): warning: relocation refers to discarded section
/lib/../lib64/crt1.o(.gnu.build.attributes.unlikely+0x18): warning: relocation refers to discarded section
/lib/../lib64/crt1.o(.gnu.build.attributes.unlikely+0x20): warning: relocation refers to discarded section
/lib/../lib64/crt1.o(.gnu.build.attributes.startup+0x18): warning: relocation refers to discarded section
/lib/../lib64/crt1.o(.gnu.build.attributes.startup+0x20): warning: relocation refers to discarded section
/lib/../lib64/crt1.o(.gnu.build.attributes.exit+0x18): warning: relocation refers to discarded section
/lib/../lib64/crt1.o(.gnu.build.attributes.exit+0x20): warning: relocation refers to discarded section
/usr/lib64/libc_nonshared.a(elf-init.oS)(.gnu.build.attributes.hot+0x18): warning: relocation refers to discarded section
/usr/lib64/libc_nonshared.a(elf-init.oS)(.gnu.build.attributes.hot+0x20): warning: relocation refers to discarded section
/usr/lib64/libc_nonshared.a(elf-init.oS)(.gnu.build.attributes.unlikely+0x18): warning: relocation refers to discarded section
/usr/lib64/libc_nonshared.a(elf-init.oS)(.gnu.build.attributes.unlikely+0x20): warning: relocation refers to discarded section
/usr/lib64/libc_nonshared.a(elf-init.oS)(.gnu.build.attributes.startup+0x18): warning: relocation refers to discarded section
/usr/lib64/libc_nonshared.a(elf-init.oS)(.gnu.build.attributes.startup+0x20): warning: relocation refers to discarded section
/usr/lib64/libc_nonshared.a(elf-init.oS)(.gnu.build.attributes.exit+0x18): warning: relocation refers to discarded section
/usr/lib64/libc_nonshared.a(elf-init.oS)(.gnu.build.attributes.exit+0x20): warning: relocation refers to discarded section
/rds/bear-apps/devel/eb-sjb-up/EL8/EL8-cas/software/binutils/2.30-GCCcore-7.3.0/bin/ld.gold: error: treating warnings as errors
collect2: error: ld returned 1 exit status
ninja: build stopped: subcommand failed.
Command '['/dev/shm/build-branfosj-admin/branfosj-admin-up/Qt5/5.10.1/foss-2018b/qt-everywhere-src-5.10.1/qtwebengine/src/3rdparty/ninja/ninja', '-C', '/dev/shm/build-branfosj-admin/branfosj-admin-up/Qt5/5.10.1/foss-2018b/qt-everywhere-src-5.10.1/qtwebengine/src/3rdparty/chromium/tools/gn/out/Release', 'gn']' returned non-zero exit status 1
Building gn manually in a temporary directory for bootstrapping...
Building gn using itself to out/Release...
Project ERROR: GN build error!
```

Reply:
Spelling corrected in e7badbb

Build bug fixed by #13785